app/main.py

```diff
@@ -1,6 +1,26 @@
+import os
+from datetime import datetime
+
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+URL = f"http://api.weatherapi.com/v1/current.json?key={API_KEY}&q={CITY}"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    result = requests.get(URL)
+    result.raise_for_status()
+    data = result.json()
```

Comment:
you can rewrite `data = result.json()["current"]` so it won't repeat and will be more readable